docs/section22/Rule22-12.md

```diff
@@ -0,0 +1,47 @@
+
+# CHW&CW - Rule 22-12  
+
+**Schema Version:** 0.0.25    
+**Mandatory Rule:** True  
+**Rule ID:** 22-12  
+**Rule Description:** The heat rejection loop shall be modeled with a single cooling tower  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** G3.1.3.11
+**Data Lookup:** None  
+**Evaluation Context:** Each Heat Rejection Loop  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b.
+2. B-RMR is modeled with heat rejection loop
+
+**Function Call:**  
+
+1. get_heat_rejection_loops_connected_to_baseline_systems()
+
+**Applicability Checks:**  
+
+- The function get_heat_rejection_loops_connected_to_baseline_systems() returns a list of loops that are connected to Type 7,8,12,13,7b,8b,12b.  Get the list of applicable heat rejection loops: `heat_rejection_loop_ids = get_heat_rejection_loops_connected_to_baseline_systems(B_RMI)`
+
+- check applicability for each loop in the model: `for fluid_loop in B_RMI.fluid_loops:`
+
+  - check if heat_rejection is one of the applicable loops: `if fluid_loop.id in heat_rejection_loop_ids: CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
+
+  ## Rule Logic:  
+
+  - create a heat_rejections_count variable to count the number of heat_rejections that are connected to the fluid_loop: `heat_rejections_count = 0`
+
+  - loop through the heat rejections in the model: `for heat_rejection in B_RMI.heat_rejections:`
+
+    - check if the heat_rejection is connected to the fluid_loop, and increment the heat_rejections_count if it is: `if heat_rejection.fluid == fluid_loop: heat_rejections_count += 1`
+
+
+  **Rule Assertion - FluidLoop:**
+
+  - Case 1: If there is exactly one heat rejection connected to the fluid loop, PASS: `if heat_rejections_count == 1: PASS`
```

Comment:
What if there are multiple heat rejection loops? It seems that with its current scope the RDS should be verifying that there is one cooling tower per loop, not the total number of cooling towers for all heat rejection loops.

Reply:
line 26 looks each each fluid loop.  The rule logic is indented under line 26, so applies continues the logic from the applicability check and is looking at each individual loop.